runtime/ms-rest-azure/index.d.ts

```diff
@@ -241,15 +241,18 @@ export class BaseResource {
  *
```

Comment:
Previously index.d.ts was underneath the lib directory so based on the above statement brought it at the root and also added types to package.json (as per their advise).